test/test_utils/sagemaker.py

```diff
@@ -379,6 +361,7 @@ def execute_local_tests(image, pytest_cache_params):
                             f"{pytest_command} failed with error code: {res.return_code}\n"
                             f"Traceback:\n{res.stdout}"
                         )
+        test_success = True
```

Comment:
Curious - do you know why the code below is catching every exception and just printing it instead of raising? The issue I see is that all the errors raised in the try block above, including ones we explicitly raise, are just printed out

Reply:
This is intentional. We do not want to raise Exceptions in functions called through `map` or `starmap`.

Reason - https://github.com/aws/deep-learning-containers/pull/2437#discussion_r1029936057